### DISCUSSION TITLE
'ClassificationExperiment' Error

### DISCUSSION THREAD

**Discussion by Pavan-628** [pycaret/pycaret#3657] [Q&A]
Hello All,
 I am new to pycaret, trying to learn MLOps with pycaret with MLFlow, but stuck with the following error while setting up the experiment.

_exp01 = setup(data=data, target="Churn", session_id=123, ignore_features=["customerID"], 
                numeric_features=["SeniorCitizen"],normalize=True, 
              feature_selection=True,log_experiment = True, experiment_name = 'churn_03',
              log_plots=True, log_profile=True)_

Above code throws the error  "'ClassificationExperiment' object has no attribute 'report'

Has anybody faced this issue

**Comment by kretes** [comment]
Same for me when I'm using `log_experiment="mlflow" and log_profile=True`
